### ISSUE TITLE
Support ACH ('us_bank_account') payment method

### ISSUE THREAD

**Issue by sir-captainmorgan21** [capacitor-community/stripe#296]
**Is your feature request related to a problem? Please describe.**
I am able to easily get this plugin working with the card payment method, but using the `us_bank_account` method doesn't work. The sheet opens and closes right away.

**Describe the solution you'd like**
I'd like to be able to use the us_bank_account payment method when setting up the intent

**Describe alternatives you've considered**
Haven't considered any

**Additional context**
None

**Usage Product**
If you adopt this plugin, please let us know which product you are using.

Product Name: Palmetto
Product URL : home.palmetto.com
Using Function:
- [ ] Payment Sheet / Payment Flow
- [ ] Apple Pay
- [ ] Google Pay
- [ ] Identity Verification Sheet

**Comment by sir-captainmorgan21** [comment]
We are looking into this plugin because we are having issues with using stripe-js-react in our ionic app and the user being redirected back to the native app once the ACH process is done

**Comment by sir-captainmorgan21** [comment]
Hi,

I'd love to contribute here, but the CONTRIBUTING docs dont have anything on testing or running locally. Can someone help me out here?

**Comment by sir-captainmorgan21** [comment]
Sorry to ping you directly, but would love to contribute this solution @hideokamoto-stripe would you be able to help out here. I've opened an issue over in stripe-js here: https://github.com/stripe/stripe-js/issues/463

@brendanm-stripe has been helping over there. Updating this plugin to support ACH seems to be the best solution

**Comment by sir-captainmorgan21** [comment]
It looks like its relatively simple to add support for it: https://stripe.com/docs/payments/accept-a-payment?platform=ios&ui=payment-sheet#ios-ach I just cant figure out how to test my changes locally in this repo

**Comment by sir-captainmorgan21** [comment]
@rdlabo hoping to bring this to your attention as well. Could you help me get setup with testing my changes locally?

**Comment by rdlabo** [comment]
This issue is closed because it is a duplicate of https://github.com/capacitor-community/stripe/issues/88.
